docs/app/src/docs.js

```diff
@@ -50,18 +52,23 @@ angular.module('DocsController', ['currentVersionData'])
     } else {
       $scope.currentArea = NG_NAVIGATION['api'];
       $scope.breadcrumb = [];
-      $scope.partialPath = 'Error404.html';
+      $scope.partialPath = errorPartialPath;
     }
   });
 
+  $scope.hasError = function() {
+    return $scope.partialPath === errorPartialPath || $scope.loadingError;
+  };
+
   /**********************************
    Initialize
    ***********************************/
 
   $scope.versionNumber = CURRENT_NG_VERSION.full;
   $scope.version = CURRENT_NG_VERSION.full + ' ' + CURRENT_NG_VERSION.codeName;
-  $scope.loading = 0;
-
+  $scope.loading = false;
+  $scope.loadingError = false;
+  var errorPartialPath = 'Error404.html';
```

Comment:
I know it will be hoisted but I would much prefer to find this at the top of the function.